test/stresstest/README.md

```diff
@@ -0,0 +1,11 @@
+The stresstest can be invoked by running: ''' python3 stresstest.py '''
+Before starting the stresstest run either: ''' python3 broker.py ''' or ''' python3 server.py '''
+For the configuration which kind of stresstest is executed , you can specify a config file like [config.ini](config.ini).
+In the configuration file you can specify:
+* port = port which gets used to connect the stress clients
+* ip = ip address which gets used to connect the stress clients
+* protocol = if the client should call websocket or gRPC's
```

Comment:
Mention the actual string to use, maybe something like:

protocol = if the client should call websocket (`ws`, only supported by server)  or gRPC's (`grpc`)